magento-research/pwa-studio
# [doc] Update modular components docs

## Description

Update modular component docs to remove mention of using NPM instead of Yarn.
Linting issues for that topic has also been fixed in this PR.

## Related Issue

Closes #907 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run linting: `npm run lint src/venia-pwa-concept/features/modular-components/index.md`
3. Verify no linting issues found for that topic.

## Checklist:
